fv/bpf_test.go

```diff
@@ -1899,6 +1899,12 @@ func describeBPFTests(opts ...bpfTestOpt) bool {
 					})
 
 					It("should have connectivity from a workload to a service with multiple backends", func() {
+						// FIXME we can only do the test with regular NAT as
+						// cgroup shares one random affinity map
+						if testOpts.connTimeEnabled {
+							return
```

Comment:
Please can you wrap the `It` in this `if { ... }` instead; skipping a test in this way means that we do about 30s of set-up (creating containers and workloads) before the skip so it's very inefficient.

Reply:
Ohhh sure, needs to be outside :+1: